sql/types/strings.go

```diff
@@ -41,7 +42,7 @@ const (
 	TinyTextBlobMax   = charBinaryMax
 	TextBlobMax       = varcharVarbinaryMax
 	MediumTextBlobMax = 16_777_215
-	LongTextBlobMax   = int64(4_294_967_295)
+	LongTextBlobMax   = int64(math.MaxInt32)
```

Comment:
This has been set to MaxInt32 because on 32-bit platforms, the `scanner.Buffer` takes in `int`, so for 32-bit platforms this has to be `math.MaxInt32` or else it overflows this limit. This might be an issue on existing 64-bit systems. as the value has been changed from `math.MaxUint32`